iphone/Maps/UI/Editor/AdditionalNames/MWMEditorAdditionalNamesTableViewController.mm

```diff
@@ -65,9 +65,13 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 {
   MWMTableViewCell * cell = [tableView dequeueReusableCellWithIdentifier:@"ListCellIdentifier"];
   NSInteger const index = indexPath.row;
-  StringUtf8Multilang::Lang const & language = m_languages[index];
-  cell.textLabel.text = @(language.m_name.c_str());
-  cell.detailTextLabel.text = @(language.m_code.c_str());
+  StringUtf8Multilang::Lang const & lang = m_languages[index];
+  cell.textLabel.text = [[NSString alloc] initWithBytes:lang.m_name.data()
+                                                 length:lang.m_name.size()
+                                               encoding:NSUTF8StringEncoding];
+  cell.detailTextLabel.text = [[NSString alloc] initWithBytes:lang.m_code.data()
+                                                       length:lang.m_code.size()
+                                                     encoding:NSUTF8StringEncoding];
```

Comment:
Add helper function NSString from string_view and use here and below?

Reply:
It was my initial implementation but you complained about inlining )

Reply:
I have a better idea, let's make a generic NSString extension to create it from string_view.

Reply:
Haha you already did it in ToNSString :)

Reply:
I complained about useless var instead of using function call in-place. Ok here.